websites/W/wumpus.store/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "347077478726238228",
+		"name": "Dominik"
+	},
+	"service": "wumpus.store",
+	"description": {
+		"en": "Rich Presence for wumpus.store - The #1 Discord Bot Store",
+		"de": "Rich Presence für wumpus.store - Der #1 Discord Bot Store"
+	},
+	"url": "wumpus.store",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/QCtoeEM.png",
```

Comment:
[Logo](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px